Calculator/calculator.c

```diff
@@ -0,0 +1,99 @@
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+#include<stdlib.h>
+
+#define MaxSize 100
+
+int prec(char c) {
```

Comment:
What is prec? what is c? Follow proper naming conventions at all places in the file.

Reply:
I will rectify this and use proper naming conventions.